app/jobs/send_update_profile_reminder_mail_job.rb

```diff
@@ -0,0 +1,19 @@
+class SendUpdateProfileReminderMailJob < CronJob
+  self.cron_expression = '0 3 * * 7'
+
+  def perform
+    Person.find_each do |person|
+      company = Company.find(person.company_id)
+      if person.reminder_mails_active && company.reminder_mails_active && needs_reminder?(person)
+        ReminderMailer.update_user_reminder_email(person).deliver_now!
```

Comment:
So I mean we only will be sending a few mails at a time...
But for scaling's sake you might want to use `deliver_later` ? Would allow us to take advantage of our existing delayed_job infrastructure.